src/environments/environment.ts

```diff
@@ -5,9 +5,10 @@
 
 export const environment = {
   production: false,
-  API_URL: 'http://localhost:55850/api/v1',
-  SERVER_URL: 'http://localhost:52431',
+  API_URL: 'https://swag-api.lighthousesoftware.com/api/v1',
```

Comment:
I think the default environement.ts should point and localhost and there should be a separate enviroment.ts for QA configurations